stopcovid/dialog/engine.py

```diff
@@ -100,6 +109,18 @@ def execute(
                 f"who hasn't validated or has opted out."
             )
             return []
+
+        if self.drill_instance_id:
+            return [
+                DrillStarted(
+                    phone_number=self.phone_number,
+                    user_profile=dialog_state.user_profile,
+                    drill=self.drill,
+                    first_prompt=self.drill.first_prompt(),
+                    drill_instance_id=self.drill_instance_id,
```

Comment:
Where exactly does the uuid generation happen in today's world?

Reply:
Great question; took me a while to find that myself. T[he `StartDrill` event has an optional `drill_instance_id` argument with a default UUID](https://github.com/opus-training/dialog-engine/blob/master/stopcovid/dialog/models/events.py#L59); since we're not currently passing in UUIDs to that event, it gets created there